### PR TITLE
[MRG + 1] Fix skip test lz4 not installed

### PR DESCRIPTION
**Pull request by tomMoral**

### CODE REVIEW

File: joblib/test/test_numpy_pickle.py [hunk 1]
```diff
@@ -547,6 +547,10 @@ def test_joblib_compression_formats(tmpdir, compress, cmethod):
                 numpy_pickle.dump(obj, dump_filename,
                                   compress=(cmethod, compress))
             excinfo.match(msg)
+        elif cmethod == 'lz4' and with_lz4.args[0]:
+            # Skip the test if lz4 is not installed. We here use the with_lz4
+            # skipif fixture whose argument is True when lz4 is not installed
+            raise SkipTest("lz4 is not installed.")
```
Comment (aabadie) [inline joblib/test/test_numpy_pickle.py]: This line is not covered by the CI now.

Comment (tomMoral) [reply joblib/test/test_numpy_pickle.py]: Ok it should be covered with this commit

**Comment by ogrisel** [comment]
The coverage diff highlights lines that cannot be made covered in a deterministic manner and are unrelated to LZ4.

**Comment by tomMoral** [comment]
And the diff coverage is low because the build without `lz4` do not have coverage support.

File: continuous_integration/travis/install.sh [hunk 2]
```diff
@@ -69,19 +69,23 @@ create_new_conda_env() {
 
 create_new_conda_env
 
+# Install py.test timeout to fasten failure in deadlocking tests
+PIP_INSTALL_PACKAGES="pytest-timeout"
```
Comment (aabadie) [inline continuous_integration/travis/install.sh]: a bit unrelated

**Comment by aabadie** [comment]
>  the diff coverage is low because the build without lz4 do not have coverage support.

Maybe skipping lz4 with python 3.5 was not a good idea. I'm fine with an extra line in the matrix.

File: continuous_integration/travis/install.sh (hunk 2 above)
Comment (tomMoral) [reply continuous_integration/travis/install.sh]: I just refactored all the pip install in one so I had to put this one in the list.
I think it makes think easier to read but I can put it directly in the `pip install` if you prefer.

**Pull request merged**